azurerm/resource_arm_container_group.go

```diff
@@ -34,11 +34,11 @@ func resourceArmContainerGroup() *schema.Resource {
 			"ip_address_type": {
 				Type:             schema.TypeString,
 				Optional:         true,
-				Default:          "Public",
+				Default:          "public",
```

Comment:
Please assign it with API spec: https://github.com/Azure/azure-rest-api-specs/blob/master/specification/containerinstance/resource-manager/Microsoft.ContainerInstance/stable/2018-06-01/containerInstance.json#L1054